RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -85,6 +85,12 @@ public final class PaywallViewController: UIViewController {
         self.hostingController.view.frame = self.view.bounds
     }
 
+    public override func viewDidDisappear(_ animated: Bool) {
+        if isBeingDismissed {
```

Comment:
Explicit self so it's clear this isn't a global:
```suggestion
        if self.isBeingDismissed {
```